spec/factories.rb

```diff
@@ -280,7 +280,7 @@
     after(:create) { |c| c.set_preference(:per_kg, 0.5); c.save! }
   end
 
-  factory :order_with_totals_and_distribution, :parent => :order do #possibly called :order_with_line_items in newer Spree
+  factory :order_with_totals_and_distribution, :parent => :order do
```

Comment:
can we dehashrocketize it now that we're at it?

Reply:
ah, yes, but only this line please, otherwise I'll be creating merging issues all over...

Reply:
Or you can run rubocop with the autofix argument to all factories afterwards :trollface: